spec/api-browser-window-spec.js

```diff
@@ -839,6 +839,14 @@ describe('BrowserWindow module', () => {
     })
   })
 
+  describe('BrowserWindow.openDevTools()', () => {
+    it('does not crash for frameless window', () => {
+      w.destroy()
+      w = new BrowserWindow({ show: false })
```

Comment:
hm... this test doesn't actually open a frameless window. should this be `frame: false`?

Reply:
yeah, I forgot to add `frame: false`.